packages/chart/src/CdcChartComponent.tsx

```diff
@@ -1100,7 +1100,13 @@ const CdcChart: React.FC<CdcChartProps> = ({
     setSharedFilterValue,
     svgRef,
     tableData: filteredData || excludedData,
-    transformedData: clean(filteredData || excludedData),
+    transformedData: clean(
+      Array.isArray(state.brushData) && state.brushData.length
```

Comment:
@Atash3000 won't this always be true? Do you have a way of checking if the brush chart is toggled on maybe?